src/Avalonia.Controls/Platform/Win32Properties.cs

```diff
@@ -14,7 +15,7 @@ namespace Avalonia.Controls
     /// <summary>
     /// Set of Win32 specific properties and events that allow deeper customization of the application per platform.
     /// </summary>
-    public static class Win32Properties
+    public class Win32Properties
```

Comment:
Should be kept `static` and use the `AddOwner` overload taking a `Type owner` parameter instead of `<TOwner>` for `NonClientHitTestResultProperty`.

Alternatively, add a private constructor instead so we don't add an unnecessary public API.

Reply:
Forgot that we have this AddOwner overload. Switched back to static class.